core/startos/src/net/net_controller.rs

```diff
@@ -209,6 +209,33 @@ impl NetService {
             .await?;
         self.update(id, host).await
     }
+    pub async fn clear_bindings(&mut self) -> Result<(), Error> {
+        let package_id = &self.id;
+        let old_hosts = self
+            .net_controller()?
+            .db
+            .mutate(|db| {
+                let old_hosts = db
+                    .as_public()
+                    .as_package_data()
+                    .as_idx(package_id)
+                    .or_not_found(package_id)?
+                    .as_hosts()
+                    .de()?;
+                db.as_public_mut()
+                    .as_package_data_mut()
+                    .as_idx_mut(package_id)
+                    .or_not_found(package_id)?
+                    .as_hosts_mut()
+                    .ser(&Default::default())?;
+                Ok(old_hosts)
+            })
+            .await?;
+        for (id, host) in old_hosts.0 {
+            self.update(id, host).await?;
+        }
+        Ok(())
+    }
```

Comment:
yeah, still not right. looking for foreach host, set bindings to empty. then call `update` on all the *resulting* hosts

Reply:
I'm going to just do this todo, I really don't need it